doc/examples/tutorial_creditcardfraud.md

```diff
@@ -0,0 +1,110 @@
+# The Credit Card Fraud Detection Example
+
+The `creditcard_samples.csv` file in this directory comes from [Kaggle](https://www.kaggle.com/mlg-ulb/creditcardfraud). To train the model using the full dataset, you need to download the dataset manually and load the dataset into MySQL under below instructions.
+
+## Import the Dataset into MySQL
+
+Below SQL statements will create a table used to store the dataset:
+
+```sql
+%%sqlflow
+CREATE DATABASE IF NOT EXISTS creditcard;
+CREATE TABLE IF NOT EXISTS creditcard.creditcard (
+    time INT,
+	v1 float,
+	v2 float,
+	v3 float,
+	v4 float,
+	v5 float,
+	v6 float,
+	v7 float,
+	v8 float,
+	v9 float,
+	v10 float,
+	v11 float,
+	v12 float,
+	v13 float,
+	v14 float,
+	v15 float,
+	v16 float,
+	v17 float,
+	v18 float,
+	v19 float,
+	v20 float,
+	v21 float,
+	v22 float,
+	v23 float,
+	v24 float,
+	v25 float,
+	v26 float,
+	v27 float,
+	v28 float,
+	amount float,
+	class INT);
+```
+
+Then we could run a python script to insert values to that table:
+
+```python
```

Comment:
How about converting the CSV file into sql file and put it on `/docker-entrypoint-initdb.d/` so that users don't need to run the Python script in the tutorial.

Reply:
I prefer use csv to keep the same format as user downloaded. The file in the repo only contains several lines.